src/AuthLDAP.php

```diff
@@ -3342,10 +3342,8 @@ public static function importUserFromServers($options = [])
             $params['method'] = self::IDENTIFIER_EMAIL;
         }
 
-        $auth->user_present = $auth->userExists($options);
```

Comment:
`$auth->user_present` is probably read somewhere else. The code that was setting its value should be preserved.

Reply:
No, it's not used. `$auth` is declared locally, and the only other usage is to call `getAuthMethods()` which is a shortcut for  a `getAllDataFromTable('glpi_authldaps')`.

Reply:
Anyway, added back in commit 46aabff47fe9e22e1ad19a8274383fcefcce7d16